src/models/__tests__/service_preference.test.ts

```diff
@@ -57,6 +60,64 @@ const aRetrievedServicePreference: RetrievedServicePreference = {
   ...aStoredServicePreference
 };
 
+describe("ServicePreference::Codec", () => {
+  it("retrocompatibility - should succeed decoding a disabled inbox ServicePreference with enabled preferences and DENY accessReadMessageStatus", async () => {
+    const aWrongServicePreference = {
```

Comment:
This should not be the wrong type
```suggestion
    const aServicePreference = {
```